elementary/monitor/data_monitoring/alerts/integrations/slack/slack.py

```diff
@@ -873,6 +878,147 @@ def _get_group_by_table_template(
         )
 
+    def _add_compact_sub_group_details_block(
+        self,
+        details_blocks: list,
+        alerts: Sequence[
+            Union[
+                TestAlertModel,
+                ModelAlertModel,
+                SourceFreshnessAlertModel,
+                GroupedByTableAlerts,
+            ],
+        ],
+        sub_title: str,
+        bullet_icon: str,
+    ) -> None:
+        if not alerts:
+            return
+        details_blocks.append(
+            self.message_builder.create_text_section_block(
+                f":{bullet_icon}: {len(alerts)} {sub_title}"
+            )
+        )
+
+    def _get_compact_sub_group_details_block(
+        self, alert: GroupedAlert, *args, **kwargs
+    ) -> List[dict]:
+        details_blocks: List[dict] = []
+        self._add_compact_sub_group_details_block(
+            details_blocks=details_blocks,
+            alerts=alert.model_errors,
+            sub_title="Model Errors",
+            bullet_icon="X",
+        )
+        self._add_compact_sub_group_details_block(
+            details_blocks=details_blocks,
+            alerts=alert.test_failures,
+            sub_title="Test Failures",
+            bullet_icon="small_red_triangle",
+        )
+        self._add_compact_sub_group_details_block(
+            details_blocks=details_blocks,
+            alerts=alert.test_warnings,
+            sub_title="Test Warnings",
+            bullet_icon="warning",
+        )
+        self._add_compact_sub_group_details_block(
+            details_blocks=details_blocks,
+            alerts=alert.test_errors,
+            sub_title="Test Errors",
+            bullet_icon="exclamation",
+        )
+        return details_blocks
+
+    def _get_grouped_compact_template(
+        self, alert: GroupedAlert
+    ) -> SlackAlertMessageSchema:
+        self.message_builder.add_message_color(self._get_color(alert.status))
+
+        title_blocks = [
+            self.message_builder.create_header_block(
+                f"{self._get_display_name(alert.status)}: {alert.summary}"
+            )
+        ]
+
+        details_blocks = self._get_compact_sub_group_details_block(alert)
+        return SlackAlertMessageSchema(title=title_blocks, details=details_blocks)
+
+    def _add_sub_group_details_block(
+        self,
+        details_blocks: list,
+        alerts: Sequence[
+            Union[
+                TestAlertModel,
+                ModelAlertModel,
+                SourceFreshnessAlertModel,
+                GroupedByTableAlerts,
+            ],
+        ],
+        sub_title: str,
+        bullet_icon: str,
+    ) -> None:
+        if not alerts:
+            return
+
+        section_text_rows = [f"*{sub_title}*"]
+        for alert in alerts:
+            text = f":{bullet_icon}: {alert.summary}"
+            if alert.report_url:
+                text = " - ".join([text, f"<{alert.report_url}|View Details>"])
```

Comment:
check the url in your alerts, I think the report_url tasks to the dashboard, is this what we want?

Reply:
Done.